brion/blueConfig.cpp

```diff
@@ -332,8 +339,7 @@ std::string BlueConfig::getCircuitTarget() const
 
 GIDSet BlueConfig::parseTarget( const std::string& target ) const
 {
-    return brion::Target::parse(
-        getTargets(), target.empty() ? _impl->getCircuitTarget() : target );
+    return brion::Target::parse( getTargets(), target );
```

Comment:
This change seems reasonable and we should fix according code respectively. If one wants to parse the circuit target, it should be explicitly called like that.